assets/aml-benchmark/components/batch_benchmark_inference_claude/spec.yaml

```diff
@@ -0,0 +1,212 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
+type: pipeline
+
+name: batch_benchmark_inference_claude
+display_name: Batch Benchmark Inference with claude support
+description: Components for batch endpoint inference
+version: 0.0.1
+
+inputs:
+  input_dataset:
+    type: uri_folder
+    description: Input jsonl dataset that contains prompt.  For the performance test, this one will be neglected.
+    optional: True
+  model_type:
+    type: string
+    description: Type of model. Can be one of ('aoai', 'oss', 'vision_oss, 'claude')
```

Comment:
There is a missing `'` after vision_oss.

Reply:
Fixed